src/MapContext.ts

```diff
@@ -1,8 +1,10 @@
 import React from 'react'
+import { Map } from 'leaflet'
 
-const value: any = {
-  layers: [],
-  mapRef: null,
+export type MapInstance = Map | null
```

Comment:
I think using this type might be a bit confusing for consumers of this library. If the consumer sees the type as below they might not see that this object could be null. I think it would be better if we just use the type `Map | null` directly so that it's visible.

![image](https://user-images.githubusercontent.com/695720/73265232-cc056800-41d4-11ea-865c-050ac49e410f.png)